pkgs/games/etlegacy/default.nix

```diff
@@ -0,0 +1,124 @@
+{ stdenv
+, lib
+, makeWrapper
+, writeScriptBin
+, fetchgit
+, fetchFromGitHub
+, fetchzip
+, fetchurl
+, runCommand
+, cmake
+, git
+, glew
+, SDL2
+, zlib
+, minizip
+, libjpeg
+, curl
+, lua5_4
```

Comment:
here it's best to pass `lua` and pass the lua version in callPackage, e.g., `callPackage ./default.nix { lua = lua5_4; }`

Reply:
yes please.

Reply:
@teto , I noticed that when passing in `lua`, I had to use specify `pkgsi686Linux.lua5_4` in order to build the 32-bit version (which most people play) with `pkgs.callPackage_i686`.

Reply:
I've pushed a new commit which includes passing in `lua` as agreed.

Reply:
the committed version doesn't specify lua so we have.
```
> CMake Error at /nix/store/ir1ws5v6q9b4qpky3y48y9a9lmzg5q0c-cmake-3.25.2/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
       >   Could NOT find Lua: Found unsuitable version "5.2.4", but required is at
       >   least "5.4" (found
```       
also it used the common callPackage so maybe you forgot to push these changes. Once fixed, I'll merge.

Reply:
Oops! Thank you. I've fixed that but perhaps just hold off on merging a little longer as I'm just coordinating with someone on a Darwin system to see if we can get the build working for that arch also.